### PR TITLE
fix typos in GitHub issue bug report template

### PR DESCRIPTION
**Pull request by tyrann0us**
They have been flagged by Grammarly.

### CODE REVIEW

**Comment by jurre** [comment]
Going to let a native speaker on the team take a look at these but seems like style changes and they seem to make things _less_ clear to me

**Comment by tyrann0us** [comment]
Thanks. Yes, the only actual grammar fix is
> […] to reproducing […]

to
> […] to reproduce […]

The rest are "style" changes.

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 1]
```diff
@@ -18,18 +18,18 @@ assignees: ''
 <!-- If applicable, specify the package manager version you're using (e.g., npm 7.1, pip-compile 5.0, etc.) -->
 **Language version**
 <!-- If applicable, specify the language version you're using (e.g., node 14.1, Ruby 2.7, etc. ) -->
-**Manifest location and content prior to update**
+**Manifest location and content before update**
```
Comment (brrygrdn) [inline .github/ISSUE_TEMPLATE/bug-report.md:21]:
```suggestion
**Manifest location and content before the Dependabot update**
```

👍🏻  on the before/prior to change - 'the update' seems ambiguous when I read it back so let's be specific.

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 2]
```diff
@@ -18,18 +18,18 @@ assignees: ''
...
 **Updated dependency**
 <!-- If applicable, the dependency name and to and from versions -->
-**What you expected to see, versus what you actually saw**
+**What you expected to see, versus what you saw**
 <!-- A clear and concise description of what you expected to happen -->
 **Native package manager behavior**
 <!-- If applicable, what output do you see when you update the dependency using the native package manager (e.g., bundler, npm, etc.)? -->
-**Images of the diff or a link to the PR, issue or logs**
-<!-- If applicable, add links to public PR's or Issues that Dependabot opened, and/or paste in any related logs -->
+**Images of the diff or a link to the PR, issue, or logs**
```
Comment (brrygrdn) [inline .github/ISSUE_TEMPLATE/bug-report.md:32]: Controversial, but I am a fan of the oxford comma myself 😂

**Review by brrygrdn — CHANGES_REQUESTED** [review]
I've added a couple of suggestions above, otherwise this seems like a sensible tidy up

**Pull request merged**